src/uu/cksum/src/cksum.rs

```diff
@@ -429,7 +483,8 @@ pub fn uu_app() -> Command {
             Arg::new(options::TAG)
                 .long(options::TAG)
                 .help("create a BSD style checksum, undo --untagged (default)")
-                .action(ArgAction::SetTrue),
+                .action(ArgAction::SetTrue)
+                .overrides_with(options::UNTAGGED),
```

Comment:
Semantically, yes. By all means YES! But GNU is terrible, and `--tag` actually means `--tag --text`, and therefore overrides `--binary`. And that's why this cannot work, because clap sees `--untagged` and erases any evidence that `--tag` was given.

Example: `--binary --tag --untagged` is actually equivalent to `--text --untagged`, so we end up with this abomination:

```console
$ ../gnu/src/cksum -a md5 --binary --untagged README.md 
e5773576fc75ff0f8eba14f61587ae28 *README.md
$ ../gnu/src/cksum -a md5 --binary --tag --untagged README.md 
e5773576fc75ff0f8eba14f61587ae28  README.md
```

I see two ways around this:
- Remove this particular `overrides_with`, and deal with it in code, using the same `indices_of().last()`-trick as `ls`. This definitely works, and should be the most "obvious" approach.
- Keep this override, and *add* `overrides_with(options::binary)`. I'm not sure whether clap handles this correctly, and I'm not sure whether this causes any other problems.